creator/did_gen.py

```diff
@@ -0,0 +1,106 @@
+"""Script to generate DID documents for did:web.
+
+(c) Anja Strunk <anja.strunk@cloudandheat.com>, 4/2024
+SPDX-License-Identifier: EPL-2.0
+"""
+
+import base64
+from dataclasses import dataclass
+from typing import List
+
+import requests
+from cryptography import x509
+from cryptography.hazmat.primitives import serialization
+from jwcrypto.jwt import JWK
+
+
+@dataclass
+class VerificationMethod():
+    path: str
+    x509: bool = False
+
+
+CONTEXT = [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/jws-2020/v1"]
+
+
+def generate_did_document(issuer: str, verification_methods: List[VerificationMethod]) -> dict:
+    """ Return a DID document for given issuer and with given  verification methods as dict.
+
+    :param issuer: did:web of issuer
+    :type issuer: str
+    :param verification_methods: List of verification method to be added to DID document.
+    :type verification_methods: List of VerificationMethods
+    :@return: DID document as dict
+    :rtype dict
+    """
+
+    did_doc = dict()
+    did_doc['@context'] = CONTEXT
+    did_doc['id'] = issuer
+    did_doc['verificationMethod'] = list()
+    did_doc['assertionMethod'] = list()
+
+    key_number = 0
+    for m in verification_methods:
```

Comment:
```suggestion
    for key_number, m in enumerate(verification_methods):
```

(the pythonic way :tm:)